### DISCUSSION TITLE
Last updated date as variable to insert in summary?

### DISCUSSION THREAD

**Discussion by gitttt54** [meisnate12/Plex-Meta-Manager#1263] [Ideas]
Hi,

I'd like to know, if there is any way to add a text, say "(last updated at dd.mm.yyyy)" into the summary of collections when they're updated by TMM.

This info should be added to inform the users about the last date, the collection was updated by the "plex server", 
or, for me as an admin,
to easily see that TMM (or trakt or tvdb....) had an error/problem with updating a collection, when I'm not at the console...

something like a variable which we can put in the summary, like 
`
    summary: Movies That Everyone is Watching (last updated at '+$variable+')'
`

Thank you

**Comment by meisnate12** [comment]
you should add this to https://features.metamanager.wiki/
